store/type-map.js

```diff
@@ -218,6 +220,11 @@ export function DSL(store, product, module = 'type-map') {
     virtualType(obj) {
       store.commit(`${ module }/virtualType`, {product, obj});
     },
+
+    spoofedType(obj) {
+      console.log('spoofing', product, obj);
```

Comment:
Errant console.log  (This doesn't show up because eslint is disabled on this file for unrelated reasons; it crashes on the import{List,Detail,Edit} methods)

Reply:
Removed